waiter/src/waiter/main.clj

```diff
@@ -117,6 +118,7 @@
     (reify Thread$UncaughtExceptionHandler
       (uncaughtException [_ thread throwable]
         (log/error throwable (str (.getName thread) " threw exception: " (.getMessage throwable))))))
+  (jvm-metrics/instrument-jvm)
```

Comment:
I haven't measured the performance impact but wouldn't expect these to be high. The jvm metrics are all gauges, which means they only execute when we ask for the value.